content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -14,7 +15,22 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ```bash
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm install prometheus prometheus-community/prometheus --namespace <PROMETHEUS_NS>
+helm install prometheus prometheus-community/prometheus --namespace default
+```
+
+- The *prometheus-service* needs access to a Prometheus instance. If you have completed the steps from [Setup Prometheus](./#setup-prometheus), the *prometheus-service* uses the Prometheus instance running in the cluster. Otherwise,
```

Comment:
Can you please change this paragraph to: 

The *prometheus-service* needs access to a Prometheus instance and by default uses the Prometheus instance running in the cluster. If another Prometheus instance shall be used, create a *secret* containing the **user**, **password**, and **url**. The secret must have the following format (please note the double-space indentation):

Reply:
Done